schemas.yaml

```diff
@@ -574,6 +563,76 @@ schemas:
           ipAddress: 192.168.1.13
           subnetMask: 255.255.255.0
           standardGateway: 192.168.1.1
+
+  voltageObject:
+    type: object
+    properties:
+      value:
+        type: number
+        minimum: 0
+        example: 24
```

Comment:
Changing the example from "24" to "24.0" indicates its type clearer.

Reply:
Please check the latest commit and resolve conservation if it fits your expectations.